packages/beacon-node/src/chain/archiveStore/utils/replayBlocks.ts

```diff
@@ -0,0 +1,101 @@
+import {PubkeyIndexMap} from "@chainsafe/pubkey-index-map";
+import {BeaconConfig} from "@lodestar/config";
+import {
+  BeaconStateAllForks,
+  CachedBeaconStateAllForks,
+  DataAvailabilityStatus,
+  ExecutionPayloadStatus,
+  createCachedBeaconState,
+  stateTransition,
+} from "@lodestar/state-transition";
+import {Slot} from "@lodestar/types";
+import {toHex} from "@lodestar/utils";
+import {IBeaconDb} from "../../../db/index.js";
+
+/**
+ * Populate a PubkeyIndexMap with any new entries based on a BeaconState
+ */
+export function syncPubkeyCache(state: BeaconStateAllForks, pubkey2index: PubkeyIndexMap): void {
```

Comment:
This is a duplicate of `syncPubkeyCache` in `getHistoricalState.ts`. Is this intentional?